rpms/build.sh

```diff
@@ -82,19 +83,16 @@ if [[ "${future}" == "yes" ]]; then
     old_name="wazuh-${build_target}-${base_version}-${package_release}"
     package_name=wazuh-${build_target}-${wazuh_version}
     old_package_name=wazuh-${build_target}-${base_version}
-    cp -r "${specs_path}/${base_version}" "${specs_path}/${wazuh_version}"
-    cd "${specs_path}/${wazuh_version}"
-    rename "${base_version}" "${wazuh_version}" *${base_version}*
-    cd -
+    cp -r "${specs_path}/" "${specs_path}/"
```

Comment:
Please verify if this is necessary.

Reply:
In fact, this copy command does not make a copy on the same folder?

Reply:
Solved [760d801](https://github.com/wazuh/wazuh-packages/pull/1571/commits/760d801a242f59dbd739a73b0bd66b8291aeaf66)